### PR TITLE
Update for the German translation file

### PR DESCRIPTION
**Pull request by ovalseven8**

### CODE REVIEW

File: translations/de.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <translation>Audio/Video</translation>
     </message>
     <message>
-        <source>%1x%2</source>
+        <source>%1p</source>
```
Comment (sudden6) [inline translations/de.ts:11]: This should probably be not translateable.

File: translations/de.ts [hunk 2]
```diff
@@ -275,6 +254,22 @@ Viele Grüße,
 %1</translation>
     </message>
     <message>
+        <source>Add a friend</source>
+        <translation>Einen Kontakt hinzufügen</translation>
```
Comment (sudden6) [inline translations/de.ts]: don't we normally use "Freund" over "Kontakt"?

File: translations/de.ts [hunk 3]
```diff
@@ -780,6 +775,14 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <translation>Chat aus diesem Fenster entfernen</translation>
     </message>
     <message>
+        <source>To new group</source>
+        <translation>Zu einer neuen Gruppe</translation>
+    </message>
+    <message>
+        <source>Invite to group &apos;%1&apos;</source>
+        <translation>Einladen in Gruppe „%1“</translation>
```
Comment (sudden6) [inline translations/de.ts]: "In Gruppe "%1" einladen" ?

File: translations/de.ts [hunk 4]
```diff
@@ -1259,6 +1262,33 @@ werden übertragen, sobald ihr beide online seid.</translation>
     </message>
 </context>
 <context>
+    <name>GroupInviteForm</name>
+    <message>
+        <source>Invited by &lt;b&gt;%1&lt;/b&gt; on %2.</source>
```
Comment (sudden6) [inline translations/de.ts:1268]: HTML removal needed

#2985

File: translations/de.ts (hunk 2 above)
Comment (ovalseven8) [inline translations/de.ts]: Not necessarily, it depends on the phrase. But I changed it. :)

File: translations/de.ts (hunk 3 above)
Comment (ovalseven8) [inline translations/de.ts]: Ok...

**Pull request merged**